domains/jishnupsamal.is-local.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Domain for my Personal Website",
+
+  "domain": "is-local.org",
+  "subdomain": "jishnupsamal",
+
+  "owner": {
+    "repo": "https://github.com/jishnu-prasad-s/",
+    "email": "jishnu@jishnupsamal.ml"
+  },
+
+  "record": {
+    "CNAME": "jishnupsamal.pages.dev.",
+    "TXT": ["google-site-verification=z8dYpXnBW_3nbymHydqToVX8_8YLEprdGtxWiPCNF4I"]
```

Comment:
You cannot have a CNAME record with other records present. 

**CNAME records must stand alone without anything else.** 

If you'd like to keep your TXT records, it's recommended you use A or AAAA records instead of CNAME.